library/core/networking/src/main/java/org/quiltmc/qsl/networking/api/ServerPlayNetworking.java

```diff
@@ -0,0 +1,298 @@
+		Objects.requireNonNull(handler, "Server play network handler cannot be null");
+
+		return ServerNetworkingImpl.getAddon(handler).getSendableChannels();
+	}
+
+	/**
+	 * Checks if the connected client declared the ability to receive a packet on a specified channel name.
+	 *
+	 * @param player the player
+	 * @param channelName the channel name
+	 * @return {@code true} if the connected client has declared the ability to receive a packet on the specified channel, otherwise {@code false}
+	 */
+	public static boolean canSend(ServerPlayerEntity player, Identifier channelName) {
+		Objects.requireNonNull(player, "Server player entity cannot be null");
+
+		return canSend(player.networkHandler, channelName);
+	}
+
+	/**
+	 * Checks if the connected client declared the ability to receive a packet on a specified channel name.
+	 *
+	 * @param handler the network handler
+	 * @param channelName the channel name
+	 * @return {@code true} if the connected client has declared the ability to receive a packet on the specified channel, otherwise {@code false}
+	 */
+	public static boolean canSend(ServerPlayNetworkHandler handler, Identifier channelName) {
+		Objects.requireNonNull(handler, "Server play network handler cannot be null");
+		Objects.requireNonNull(channelName, "Channel name cannot be null");
+
+		return ServerNetworkingImpl.getAddon(handler).getSendableChannels().contains(channelName);
+	}
+
+	/**
+	 * Creates a packet which may be sent to a the connected client.
+	 *
+	 * @param channelName the channel name
+	 * @param buf the packet byte buf which represents the payload of the packet
+	 * @return a new packet
+	 */
+	public static Packet<?> createS2CPacket(Identifier channelName, PacketByteBuf buf) {
+		Objects.requireNonNull(channelName, "Channel cannot be null");
+		Objects.requireNonNull(buf, "Buf cannot be null");
+
+		return ServerNetworkingImpl.createPlayC2SPacket(channelName, buf);
+	}
+
+	/**
+	 * Gets the packet sender which sends packets to the connected client.
+	 *
+	 * @param player the player
+	 * @return the packet sender
+	 */
+	public static PacketSender getSender(ServerPlayerEntity player) {
+		Objects.requireNonNull(player, "Server player entity cannot be null");
+
+		return getSender(player.networkHandler);
+	}
+
+	/**
+	 * Gets the packet sender which sends packets to the connected client.
+	 *
+	 * @param handler the network handler, representing the connection to the player/client
+	 * @return the packet sender
+	 */
+	public static PacketSender getSender(ServerPlayNetworkHandler handler) {
+		Objects.requireNonNull(handler, "Server play network handler cannot be null");
+
+		return ServerNetworkingImpl.getAddon(handler);
+	}
+
+	/**
+	 * Sends a packet to a player.
+	 *
+	 * @param player the player to send the packet to
+	 * @param channelName the channel of the packet
+	 * @param buf the payload of the packet
+	 */
+	public static void send(ServerPlayerEntity player, Identifier channelName, PacketByteBuf buf) {
```

Comment:
Would it be worth it to add a `send(Collection<ServerPlayerEntity>, Identifier, PacketByteBuf)` method? In many cases packets need to be sent to multiple players, and `send(PlayerLookup.tracking(e) id, buf)` is much quicker to write and easier to read than `PlayerLookup.tracking(e).forEach(p -> send(p, id, buf))`